py_bentoctl/other_helpers.py

```diff
@@ -288,6 +293,206 @@ def init_docker(client: docker.DockerClient):
             task_print_done(f"network created (name: {net_name}).")
 
 
+def stash_element_extra_properties(phenopacket: dict, element_name: str, stash: dict):
+    element = phenopacket.get(element_name, {})
+    if type(element) is list:
+        for idx, item in enumerate(element):
+            # Align with item index if no "id" in the element item
+            item_id = item.get("id", idx)
+            if extra_properties := item.pop("extra_properties", False):
```

Comment:
i prefer `None` as a default for pop rather than `False` since it matches `.get` defaults